### DISCUSSION TITLE
Release 5.3.0

### DISCUSSION THREAD

**Discussion by totocaca123** [jenkinsci/nested-data-reporting-plugin#399] [General]
@simonsymhoven 
Many updates have been performed (packages updates and dependencies).
I think it's time to make a release 5.3.0, as it is a major update to avoid this plugin to become obselete.
I tried to do it but master branch is locked

**Comment by simonsymhoven** [comment]
Hi @totocaca123, sorry I have no time at the moment to invest into the plugin development.. I removed the branch protection. thanks for your help and if you need anything please feel free to contact me.

**Reply by totocaca123** [reply]
ok thanks I will try to make a release soon

**Reply by totocaca123** [reply]
Hi @simonsymhoven 

The master branch is unlocked.  Now I use maven workflow  to make release automatically with cd workflow.
It works but cannot push release to jenkins repo. I have following error
` [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.1.3:deploy (default-deploy) on project nested-data-reporting: Failed to deploy artifacts: Could not transfer artifact io.jenkins.plugins:nested-data-reporting:pom:869.v21eb_394a_c29a_ from/to maven.jenkins-ci.org (https://repo.jenkins-ci.org/releases/): status code: 401, reason phrase:  (401) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.1.3:deploy (default-deploy) on project nested-data-reporting: Failed to deploy artifacts: Could not transfer artifact io.jenkins.plugins:nested-data-reporting:pom:869.v21eb_394a_c29a_ from/to maven.jenkins-ci.org (https://repo.jenkins-ci.org/releases/): status code: 401, reason phrase:  (401)`

Seems that no token is present in the repo
      MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
      MAVEN_TOKEN: ${{ secrets.MAVEN_TOKEN }}
